### DISCUSSION TITLE
TLS-Sidecar resources quota required

### DISCUSSION THREAD

**Discussion by rstemmler** [strimzi/strimzi-kafka-operator#5361] [Q&A] [answered]
Hi,

I want to run Strimzi on an Openshift cluster. The cluster requires a resources quota for each container. The deployment of Strimzi kafka fails due to the missing quota for the tls-sidecar container. 

Is it possible to define the resources of the tls-sidecar in kafka operator deployment file?
Is there a way to disable the deployment of tls-sidecar container?

Thanks a lot

**Comment by scholzj** [comment, marked as answer]
In general, you can configure the sidecar resources in the custom resource: https://strimzi.io/docs/operators/latest/full/using.html#type-TlsSidecar-reference ... the TLS sidecar is required to secure ZooKeeper access. The TLS sidecar is not used anymore in Kafka or ZooKeeper pods ... so you might need to go tot he right version of the docs depending on what Strimzi version you use.

**Reply by rstemmler** [reply]
Thanks for fast reply..
